components/org.wso2.carbon.identity.application.authenticator.oidc/src/main/java/org/wso2/carbon/identity/application/authenticator/oidc/OpenIDConnectAuthenticator.java

```diff
@@ -74,6 +78,9 @@
     private static Log log = LogFactory.getLog(OpenIDConnectAuthenticator.class);
     private static final String OIDC_DIALECT = "http://wso2.org/oidc/claim";
 
+    private static final String dynamicParameterLookupRegex = "\\$\\{(\\w+)\\}";
```

Comment:
change the static final variable name to all caps: `DYNAMIC_PARAMETER_LOOKUP_REGEX`